test/api_container_wait_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/alibaba/pouch/test/request"
+
+	"github.com/go-check/check"
+)
+
+// APIContainerWaitSuite is the test suite for container wait API.
+type APIContainerWaitSuite struct{}
+
+func init() {
+	check.Suite(&APIContainerWaitSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *APIContainerWaitSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	PullImage(c, busyboxImage)
+}
+
+// TestWaitOk tests waiting a stopped container return 200.
+func (suite *APIContainerWaitSuite) TestWaitOk(c *check.C) {
+	cname := "TestWaitOk"
+
+	CreateBusyboxContainerOk(c, cname)
+	StartContainerOk(c, cname)
+	StopContainerOk(c, cname)
+
+	resp, err := request.Post("/containers/" + cname + "/wait")
+	c.Assert(err, check.IsNil)
+	CheckRespStatus(c, resp, 200)
+
+	DelContainerForceOk(c, cname)
+}
+
+// TestRunningContainer tests waiting a running container to stop, then returns 200.
+func (suite *APIContainerWaitSuite) TestRunningContainer(c *check.C) {
+	cname := "TestRunningContainer"
```

Comment:
Please use the `TestWaitForRunningContainer` so that the case doesn't impact other case.

Reply:
Agree! @xiechengsheng please change the function name, because `TestRunningContainer` may has confusion, thanks a lot !